features/admin/manage_users.feature

```diff
@@ -0,0 +1,27 @@
+@manage-users
+Feature: Admin manager can manage users
+
+Scenario: Admin Manager user can log in and suspend admin users
+  Given I am logged in as the production admin-manager user
+  And I am on the /admin page
+  And I click the 'Manage users' link
+  #TODO change this email address
+  And I click the summary table Edit link for '14288@user.marketplace.team'
+  When I choose the 'Suspended' radio button
+  And I click the 'Update user' button
+  Then I see a success banner message containing '14288@user.marketplace.team has been updated.'
```

Comment:
Should there be a following step here to check that the updated user is shown with the right (new) status in the summary table?

Reply:
Done